editor/debugger/script_editor_debugger.cpp

```diff
@@ -852,6 +852,14 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, uint64_t p_thread
 			emit_signal(SNAME("remote_objects_updated"), objs);
 			emit_signal(SNAME("remote_tree_select_requested"), objs->remote_object_ids.duplicate());
 		}
+	} else if (p_msg == "request_embed_suspend_toggle") {
+		emit_signal(SNAME("embed_shortcut_requested"), EMBED_SUSPEND_TOGGLE);
+	} else if (p_msg == "request_embed_next_frame") {
+		emit_signal(SNAME("embed_shortcut_requested"), EMBED_NEXT_FRAME);
+	} else if (p_msg == "remote_node_clicked") {
+		if (!p_data.is_empty()) {
+			emit_signal(SNAME("remote_tree_select_requested"), p_data[0]);
+		}
```

Comment:
Looks like rebase error?
This no longer exists.

Reply:
You are right... sorry about that.